### DISCUSSION TITLE
mediatek hardware nat支持

### DISCUSSION THREAD

**Discussion by ttc0419** [immortalwrt/immortalwrt#1079] [General]
请问项目支持mediatek的hardware nat了么？之前看了[论坛](https://forum.openwrt.org/t/use-mtk-sdk-hwnat-driver-to-replace-hw-flow-offload/128120/4)，但是他只移植了ramips的，官方的[SDK](https://git01.mediatek.com/plugins/gitiles/openwrt/feeds/mtk-openwrt-feeds/)也只有21.02的5.4 patch。

**Comment by EternalARK** [comment]
> 请问项目支持mediatek的hardware nat了么？之前看了[论坛](https://forum.openwrt.org/t/use-mtk-sdk-hwnat-driver-to-replace-hw-flow-offload/128120/4)，但是他只移植了ramips的，官方的[SDK](https://git01.mediatek.com/plugins/gitiles/openwrt/feeds/mtk-openwrt-feeds/)也只有21.02的5.4 patch。

乖，联发科说在准备OP24 kernel6.6的驱动了。到时候一切都会好起来的。

MTK驱动支持的版本只有OP21和OP24。
